functions/go/render-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins/HelmChartInflationGenerator.go

```diff
@@ -135,12 +133,28 @@ func (p *HelmChartInflationGeneratorPlugin) runHelmCommand(
 	return stdout.Bytes(), err
 }
 
-// createNewMergedValuesFile replaces/merges original values file with ValuesInline.
-func (p *HelmChartInflationGeneratorPlugin) createNewMergedValuesFile() (
-	path string, err error) {
+// createNewMergedValuesFiles replaces/merges original values file with ValuesInline.
+func (p *HelmChartInflationGeneratorPlugin) createNewMergedValuesFiles(path string) (
+	string, error) {
+	pValues, err := ioutil.ReadFile(path)
+	if err != nil {
+		if u, err := url.Parse(path); err == nil && (u.Scheme == "http" || u.Scheme == "https") {
```

Comment:
If it's a valid URL, but the scheme is not supported, we should return an error.

If it's neither a local path nor a valid URL, we should return an error.

Reply:
done